### PR TITLE
Update license headers

### PR DESCRIPTION
**Pull request by MorrisJobke**
In preparation of #6926 to see the actual difference of that script.

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

**Review by LukasReschke — APPROVED** [review]

File: apps/dav/lib/CalDAV/Search/Xml/Filter/ParamFilter.php [hunk 1]
```diff
@@ -1,21 +1,23 @@
 <?php
 /**
- * @author Georg Ehrke <oc.list@georgehrke.com>
- *
  * @copyright Copyright (c) 2017 Georg Ehrke <oc.list@georgehrke.com>
+ *
+ * @author Georg Ehrke <georg@owncloud.com>
```
Comment (georgehrke) [inline apps/dav/lib/CalDAV/Search/Xml/Filter/ParamFilter.php]: 🙈 Can you change the default to oc.list@georgehrke.com please :)

Comment (MorrisJobke) [reply apps/dav/lib/CalDAV/Search/Xml/Filter/ParamFilter.php]: separate PR.

Comment (georgehrke) [reply apps/dav/lib/CalDAV/Search/Xml/Filter/ParamFilter.php]: ok :)

Comment (MorrisJobke) [reply apps/dav/lib/CalDAV/Search/Xml/Filter/ParamFilter.php]: https://github.com/nextcloud/server/pull/7087

**Pull request merged**